src/_pytest/skipping.py

```diff
@@ -1,4 +1,4 @@
-# coding=utf8
```

Comment:
let's use `fix-encoding-pragma` from `pre-commit/pre-commit-hooks`

(yep, there's a pre-commit hook for everything!)

Reply:
Sure, sounds good! Could you please contribute that in place of this PR then? 😁

Reply:
done!

Reply:
Can it only do it if necessary maybe?

Reply:
for consistency it's all or none, that way you don't have to think about it

Reply:
Isn't the main point of pre-commit already that "you don't have to think about it", i.e. you do not have to care, and it will insert/request it if necessary?

Reply:
yeah so now you don't have to futz with whether a file has it or not, it always has it